src/common/enums.ts

```diff
@@ -36,3 +36,8 @@ export enum IssueCredentialUserText {
 	done = 'Accepted',
 	abandoned = 'Declined'
 }
+
+export enum EcosystemRoles {
```

Comment:
Please be specific like `Ecosystem Member` or `Ecosystem Lead`

Reply:
Resolved.